package-lock.json

```diff
@@ -922,10 +922,10 @@
       "resolved": "https://registry.npmjs.org/@clearlydefined/spdx/-/spdx-0.1.0.tgz",
       "integrity": "sha512-5lmNSQk875oI9X9oAVPxedxoQIjqFMchB6pYJWz+ABvPjXZz2sa41o0nbWg5Z/793TRIny6WT/LjKXTZIfKVdQ==",
       "requires": {
-        "spdx-expression-parse": "github:clearlydefined/spdx-expression-parse.js#fork",
+        "spdx-expression-parse": "github:clearlydefined/spdx-expression-parse.js#9c9211181d82c9df70dae6e2599ee84fd19e4858",
```

Comment:
I'm on node 10.15.3 and npm 6.4.1

Reply:
I'm on node 10.16.0 and npm 6.9.0. I'll merge this PR. We'll see how it goes.

Reply:
Docker build has failed. I'll update package-lock.json and let you know.
```
...
> website@0.1.0 copy-monaco /opt/website
> node ./src/utils/install-monaco.js

Creating an optimized production build...
Failed to compile.

./node_modules/@clearlydefined/spdx/node_modules/spdx-satisfies/index.js
Cannot find module: 'spdx-ranges'. Make sure this package is installed.

You can install this package by running: npm install spdx-ranges.


npm ERR! code ELIFECYCLE‌
npm ERR! errno 1‌
npm ERR! website@0.1.0 build: `env-cmd --fallback .&#x2F;.env.production npm run build-css &amp;&amp; npm run copy-monaco &amp;&amp; react-scripts build`‌
npm ERR! Exit status 1
npm ERR! ‌
npm ERR!‌ Failed at the website@0.1.0 build script.‌
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Reply:
I had to remove package-lock.json and node_modules to resolve this problem. I've also updated the PR build to run docker build to make sure the checks would fail in the future if package-lock.json isn't good.